<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPU_VolumeReconstructionWidget</class>
 <widget class="QWidget" name="GPU_VolumeReconstructionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="usAcquisitionLabel">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>US Acquisition:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="usAcquisitionComboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Volume Spacing (isotropic): </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="usVolumeSpacingComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="usSearchRadiusLabel">
       <property name="text">
        <string>US Search Radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="usSearchRadiusComboBox">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="userFeedbackLabel">
     <property name="text">
      <string>Ready to roll.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
